_includes/templates/install/ubuntu-queue-rabbitmq.md

```diff
@@ -0,0 +1,74 @@
+#### RabbitMQ Installation
+
+Since RabbitMQ is written in Erlang, you need to install Erlang before you can use RabbitMQ:
+
+```text
+cd ~
+wget http://packages.erlang-solutions.com/site/esl/esl-erlang/FLAVOUR_1_general/esl-erlang_20.1-1~ubuntu~xenial_amd64.deb
```

Comment:
dshvaika@dshvaika:~$ wget http://packages.erlang-solutions.com/site/esl/esl-erlang/FLAVOUR_1_general/esl-erlang_20.1-1~ubuntu~xenial_amd64.deb
--2020-04-23 18:43:01--  http://packages.erlang-solutions.com/site/esl/esl-erlang/FLAVOUR_1_general/esl-erlang_20.1-1~ubuntu~xenial_amd64.deb
Resolving packages.erlang-solutions.com (packages.erlang-solutions.com)... 143.204.89.90, 143.204.89.43, 143.204.89.45, ...
Connecting to packages.erlang-solutions.com (packages.erlang-solutions.com)|143.204.89.90|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2020-04-23 18:43:01 ERROR 404: Not Found.
dshvaika@dshvaika:~$

Reply:
Not Found
The requested URL /site/esl/esl-erlang/FLAVOUR_1_general/esl-erlang_20.1-1~ubuntu~xenial_amd64.deb was not found on this server.

Reply:
done